[FEATURE] Traduction des e-mails ES & NL suppr compte (PIX-15787)

pix-service-prod opened the pull request:
- L'e-mail de suppression du compte est bien reçu en espagnol 
- L'e-mail de suppression du compte est bien reçu en néerlandais

pix-service-prod committed:
Updated translations from Phrase

phrase: update api/translations/nl.json to latest state
phrase: update api/translations/es.json to latest state

pix-bot-github commented:
Une fois les applications déployées, elles seront accessibles via les liens suivants :
  * [App (.fr)](https://app-pr10846.review.pix.fr)
  * [App (.org)](https://app-pr10846.review.pix.org)
  * [Orga (.fr)](https://orga-pr10846.review.pix.fr)
  * [Orga (.org)](https://orga-pr10846.review.pix.org)
  * [Certif (.fr)](https://certif-pr10846.review.pix.fr)
  * [Certif (.org)](https://certif-pr10846.review.pix.org)
  * [Junior](https://junior-pr10846.review.pix.fr)
  * [Admin](https://admin-pr10846.review.pix.fr)
  * [API](https://api-pr10846.review.pix.fr/api/)
  * [Audit Logger](https://pix-audit-logger-review-pr10846.osc-fr1.scalingo.io/api/)

Les variables d'environnement seront accessibles via les liens suivants :
  * [scalingo front](https://dashboard.scalingo.com/apps/osc-fr1/pix-front-review-pr10846/environment)
  * [scalingo api](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr10846/environment)
  * [scalingo audit-logger](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr10846/environment)

pix-service-prod committed:
Updated translations from Phrase

phrase: update api/translations/nl.json to latest state
phrase: update api/translations/es.json to latest state

pix-service-prod committed:
Updated translations from Phrase

phrase: update api/translations/nl.json to latest state
phrase: update api/translations/es.json to latest state

pix-service-prod committed:
Updated translations from Phrase

phrase: update api/translations/nl.json to latest state
phrase: update api/translations/es.json to latest state
